tests/utils/siteBuilder.js

```diff
@@ -1,9 +1,20 @@
 const { toToml } = require('tomlify-j0.4')
-const fs = require('fs-extra')
+const _fs = require('fs')
 const path = require('path')
 const { v4: uuidv4 } = require('uuid')
 const tempDirectory = require('temp-dir')
 const os = require('os')
+const util = require('util')
+
+const fs = {
+  mkdir: util.promisify(_fs.mkdir),
+  writeFile: util.promisify(_fs.writeFile),
+  rmdir: util.promisify(_fs.rmdir),
+}
+
+const ensureDir = async path => {
+  return fs.mkdir(path, { recursive: true })
```

Comment:
[`recursive` was added](https://nodejs.org/api/fs.html#fs_fs_mkdir_path_options_callback) in Node `10.12.0`, but we unfortunately need to support Node 8. Would [`make-dir`](https://github.com/sindresorhus/make-dir) work instead?

Reply:
yep, I will change all calls to `fs.mkdir(path, { recursive: true })` to use `make-dir` instead.